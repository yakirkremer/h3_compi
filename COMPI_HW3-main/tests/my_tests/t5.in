void main(){
    auto a = (1 + 1b) + 1b + 200 ;
    auto c = ((1 + 3) >3b);
    auto h = (3b + 10b) - 5b + 200b + 200b;
    auto f = (h + h* h) + h -h; 
    auto fint = (h / h + 2* h) + h -h; 
}