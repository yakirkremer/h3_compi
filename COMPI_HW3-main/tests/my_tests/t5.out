---end scope---
a INT 0
c BOOL 1
h BYTE 2
f BYTE 3
fint INT 4
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
